microcosm_flask/paging.py

```diff
@@ -145,7 +137,7 @@ def to_dict(self):
                 for item in self.items
             ],
             _links=self._links,
-            **self.page.to_dict()
+            **self.page.to_dict(as_str=True)
```

Comment:
When using this as part of PaginatedList to serialize results to JSON repsonse,  cast to string